rotkehlchen/api/rest.py

```diff
@@ -854,8 +854,8 @@ def edit_trade(
             trade_type: TradeType,
             amount: AssetAmount,
             rate: Price,
-            fee: Optional[Fee],
-            fee_currency: Optional[Asset],
+            fee: Fee,
```

Comment:
all the typing changes in this file are not correct. `fee` and `fee_currency` are optional. What is not optional is having one but not the other.